src/python/pants/python/pex_build_util.py

```diff
@@ -214,18 +214,16 @@ def extract_single_dist_for_current_platform(self, reqs, dist_key) -> Distributi
         :raises: :class:`self.SingleDistExtractionError` if no dists or multiple dists matched the
                  given `dist_key`.
         """
-        distributions, _transitive_requirements = self.resolve_distributions(
-            reqs, platforms=["current"]
-        )
+        distributions, self.resolve_distributions(reqs, platforms=["current"])
```

Comment:
I'm not sure what the `distributions` part is doing here? Is that leftover?

Reply:
It was supposed to be an assignment! Thank you!